CODE_OF_CONDUCT.md

```diff
@@ -1,21 +1,16 @@
-# Code of Conduct
+# The Creator's Code
 
-We are sentient.  
-To be sentient is to be limited.  
-In our limitation, we make choices that are unwise or are flawed.
+Because we are limited beings,
+we make choices that are unwise or flawed.
```

Comment:
These two lines are the strophe - the set-up.

I really love the sentiment you express since it frames the whole of the rest of the text in humility and compassion.

* I've tried to keep the language simple and easy enough for young folks to be able to engage with (important for me since I spend lots of time working with young coders).
* Simple language is good because it's usually more accessible and easy to translate.
* Such sparse language also means there's plenty of space for the ideas to breath (it's not a heavy tome or legal contract).
* I've tried to avoid more philosophical terminology (sentient etc...) in favour of something philosophical but spoken clearly.